validator/src/main.rs

```diff
@@ -83,6 +85,8 @@ enum Operation {
 
 const EXCLUDE_KEY: &str = "account-index-exclude-key";
 const INCLUDE_KEY: &str = "account-index-include-key";
+const DEFAULT_MIN_SNAPSHOT_DOWNLOAD_SPEED: f32 = 10485760_f32;
+const MAX_DOWNLOAD_ABORT: u32 = 5;
```

Comment:
Let's make the max abort count a command-line option as well

Reply:
Will do